### ISSUE TITLE
Star Ocean 5 Integrity and Faithlessness BLJM61325 Black Screen on AMD CPUS ETC

### ISSUE THREAD

**Issue by DenisNinja** [RPCS3/rpcs3#14260]
### Quick summary

please fix the RCPS3 update to work Star Ocean 5 Integrity and Faithlessness BLJM61325 on AMD CPU PCS, my processor AMD RYZEN 9 5950X, this game is not working, please fix this error, i want play this game with the english patch etc

### Details

Black Screen, bug RCPS3 need update fix for AMD processor etc

### Attach a log file

the game dont boot and stay in black screen

**Comment by YuriNator557** [comment]
Does not follow the template at all, have you asked someone with an Intel CPU to test? This game is notorious for being unstable as well

**Comment by DenisNinja** [comment]
yes i have, on intel cpu this game works normal https://www.youtube.com/watch?v=gImFJ_BHzG0

**Comment by AniLeo** [comment]
Closed and locked zero effort bug report
